corehq/apps/saved_reports/models.py

```diff
@@ -718,7 +718,14 @@ def _get_and_send_report(self, language, emails):
                         'error': er,
                     }
                 )
-                if getattr(er, 'smtp_code', None) in LARGE_FILE_SIZE_ERROR_CODES or type(er) == ESError:
+                if excel_files:
+                    message = "Unable to generate email report. Excel files are attached."
+                    send_html_email_async(title, email, message,
+                                          email_from=settings.DEFAULT_FROM_EMAIL,
+                                          file_attachments=excel_files,
+                                          smtp_exception_skip_list=LARGE_FILE_SIZE_ERROR_CODES)
```

Comment:
What's this do? Ignore emails that error due to size?

Reply:
Kind of the opposite. It tells the email task not to retry if it failed because it was too large (since retry won't help)